lib/index.js

```diff
@@ -525,15 +542,46 @@ class Anser {
                     jsonChunk.bg_truecolor = null;
                 }
             } else {
+              let colors = jsonChunk.bg.split(",").map(String);
+              if (colors.length == 1) {
+                  styles.push("color:" + jsonChunk.bg + "");
+              }
+              else {
                 styles.push("background-color:rgb(" + jsonChunk.bg + ")");
+              }
             }
         }
 
         if (jsonChunk.decoration) {
             if (use_classes) {
                 classes.push("ansi-" + jsonChunk.decoration);
-            } else if (jsonChunk.decoration === "underline" || jsonChunk.decoration === "blink") {
+            } else if (jsonChunk.decoration === "bright"
```

Comment:
Instead of doing a single if here with various if's inside, this should have been a flat group of else ifs.